src/OV/OpenVarioMenu.cpp

```diff
@@ -328,6 +328,17 @@ MainMenuWidget::Prepare([[maybe_unused]] ContainerWindow &parent,
     StartXCSoar();
   });
 
+  AddButton("Logbook", [this](){
+    CancelTimer();
+    static constexpr const char *argv[] = {
+      "/usr/bin/logbook.sh", nullptr
+    };
+
+    RunProcessDialog(UIGlobals::GetMainWindow(),
+                     UIGlobals::GetDialogLook(),
+                     "Logbook", argv);
+  });
```

Comment:
This is not acceptable and will never get merged, as I told you already several months ago.

Reply:
I removed again